addons/medical_status/functions/fnc_isInStableCondition.sqf

```diff
@@ -17,7 +17,22 @@
 
 params ["_unit"];
 
-alive _unit
-&& {!IS_UNCONSCIOUS(_unit)}
-&& {GET_WOUND_BLEEDING(_unit) == 0}
-&& {_unit call FUNC(hasStableVitals)}
+if (!alive _unit
+    || {IS_UNCONSCIOUS(_unit)}
+    || {IS_BLEEDING(_unit)}
+    || {GET_HEART_RATE(_unit) < EGVAR(medical_treatment,minHeartRatePAK)}
+) exitWith {false};
+
+private _requiredBloodVolume = DEFAULT_BLOOD_VOLUME * (EGVAR(medical_treatment,minBloodPAK) / 100);
+if (GET_BLOOD_VOLUME(_unit) < _requiredBloodVolume) exitWith {false};
+
+if (EGVAR(medical_treatment,allowPainPAK) == 0 && { IS_IN_PAIN(_unit) }) exitWith {false};
+if (EGVAR(medical_treatment,allowFracturesPAK) == 0 && { !(GET_FRACTURES(_unit) isEqualTo []) }) exitWith {false};
+
+// Require at least bandaged wounds.
+if (EGVAR(medical_treatment,requiredWoundStatePAK) == 1 && { !(count GET_OPEN_WOUNDS(_unit) == (count GET_BANDAGED_WOUNDS(_unit) + count GET_STITCHED_WOUNDS(_unit))) }) exitWith {false};
```

Comment:
I think this can fail. If you want to check if there are no open wounds, check that no open wounds with `_amountOf > 0` exist.

Reply:
Corrected in #7902  -- I don't know why, but I can no longer do a PR to @anden3 's fork. Apologies for the inconvenience -- I've not run into this before & am unsure how to correct it.

I've tested the changes without issue.

Reply:
Very strange...

Reply:
I've tried again, but every attempt just goes straight to acemod/ace3 rather than anden3. No clue. Followed the exact same process I did the first time without success. 🤷‍♂️

Reply:
@dgibso29 I think I fixed it :)

Reply:
Excellent! Thank you & apologies for the inconvenience.